ethsigner/app/src/main/java/tech/pegasys/ethsigner/CommandLineConfig.java

```diff
@@ -104,31 +104,34 @@
 
   private final PrintStream output;
 
-  public CommandLineConfig(PrintStream output) {
+  public CommandLineConfig(final PrintStream output) {
     this.output = output;
   }
 
-  private HashicorpSignerCliConfig hashicorpSignerBasedConfig;
+  private HashicorpTransactionSignerCommand hashicorpTransactionSignerConfig;
```

Comment:
this should be removed - we don't want the base command knowing about the sub commands if we can help it.

Instead, either:
1. Have the EthSignerApp create this structure of commands/subcommands
2. Have EthSIgnerApp pass the BaseCommand into each SubCommand?

Reply:
I'm not sure. I like it the way it is. The config is the thing that has all the information. EthSignerApp for me is not the right place to do that.